Source/Scene/Material.js

```diff
@@ -693,6 +693,7 @@ define([
     // Create the czm_getMaterial method body using source or components.
     function createMethodDefinition(material) {
         var components = material._template.components;
+        var isMultiMaterial = material._template.materials.length > 0;
```

Comment:
`material._template.materials` is an object so this needs to be `Object.keys(material._template.materials.length).length > 0;`